src/MyParcelComApi.php

```diff
@@ -817,16 +814,34 @@ protected function getResourceUri($resourceType, $id = null)
      * @param array $array
      * @return array
      */
-    private function arrayToFilter(array $array)
+    private function arrayToFilters(array $array)
```

Comment:
Can you try to rewrite this and the next method? I have no idea what's going on. Do we really need two methods?